packages/strapi-plugin-content-manager/admin/src/components/InputUID/Options/wrapper.js

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+
+const wrapper = styled.div`
+  position: absolute;
+  background-color: white;
+  border: 1px solid ${props => props.theme.main.colors.border};
+  width: 100%;
```

Comment:
Can your order your css properties as follows:

1. Layout Properties (position, float, clear, display)
2. Box Model Properties (width, height, margin, padding)
3. Visual Properties (color, background, border, box-shadow)
4. Typography Properties (font-size, font-family, text-align, text-transform)
5. Misc Properties (cursor, overflow, z-index

I know that some components don't follow this pattern but let's try to apply whenever we can.

Reply:
Thanks